### DISCUSSION TITLE
Fonts?

### DISCUSSION THREAD

**Discussion by completewebco** [tailpress/tailpress#38] [Q&A]
What would be the best way to call the fonts in tailpress.json?

**Comment by jeffreyvr** [comment]
Hi @completewebco! Not sure what you mean. What is it that you are trying to do?

**Reply by completewebco** [reply]
Typically I define fonts in tailwind.js like this:

```
fontFamily: {
            base: ['Cera Pro', 'sans-serif'],
            heading: ['Cera Pro', 'sans-serif'],
        },
```
How would you suggest I set this in your tailpress.json file and call this from tailwind.config.js?

Great starter theme BTW!

**Reply by jeffreyvr** [reply]
Thanks! 

I would keep defining it as you do now. That's how I do it too. The tailpress.json is only used for font sizes and theme colors which also need to be available within Gutenberg/block editor.
